lib/nvme/nvme_ctrlr.c

```diff
@@ -1378,6 +1378,16 @@ spdk_nvme_ctrlr_get_data(struct spdk_nvme_ctrlr *ctrlr)
 	return &ctrlr->cdata;
 }
 
+union spdk_nvme_csts_register spdk_nvme_ctrlr_get_regs_csts(struct spdk_nvme_ctrlr *ctrlr)
```

Comment:
Just for future reference, it would be helpful if this had been split into two commits: introduce spdk_nvme_ctrlr_get_regs_csts(), then add the blockdev_nvme code that uses it.